packages/nodejs/src/util.ts

```diff
@@ -37,3 +38,12 @@ export function unwrapJsonResponse<T>(
 
     return JSON.parse(jsonString, reviver);
 }
+
+/**
+ * Loads the module as a buffer, given the given filePath.
+ * @param filepath the location of the module
+ * @returns the module as a buffer
+ */
+export function getModuleBuffer(filePath: string): Buffer {
```

Comment:
I am not sure that I understand the need for this method.

Reply:
We will keep it there, but in general I would advise users to code their own file system handling.